### ISSUE TITLE
Unverified transactions

### ISSUE THREAD

**Issue by jsblokland** [litecoin-foundation/LoafWallet#24]
Hi there,

I downloaded the LoafWalle up just yesterday I send LTC from my desktop. However the transactions are unverified, but have not returned to the Desktop app. I can't do transactions as they are not varified in the chain. Can you help me in what to do?

Thanks in advance

Jeroen Blokland

**Comment by losh11** [comment]
What Litecoin Wallet did you use to send your Litecoin on your Desktop?

**Comment by jsblokland** [comment]
Litecoin-QT, older version

**Comment by losh11** [comment]
Please update to Litecoin 0.13.2.1 from https://litecoin.com first.
After updating you should be able to send your Litecoin to your LoafWallet iOS wallet.

Your transaction is getting rejected because it's creating txs without a low-s signature. It might take a while before you're actually able to spend your Litecoin from the Litecoin Core wallet again. At max 2-3 days.

I believe that it is possible to do something to reject the incorrect transaction through console, but I can't remember at the time. Maybe take a look.

**Comment by jsblokland** [comment]
Ok I will do that. Thanks for responding so quickly. If I understand correctly the litecoins will come back eventually right? If so that is a great relief for now.

Cheers



Verstuurd vanaf mijn iPhone

**Comment by jsblokland** [comment]
Still no luck! Some of the transactions got verified but not confirmed, others are still on unverified. What to do now?

**Comment by losh11** [comment]
Do you have transaction IDs for the unconfirmed transactions?

**Comment by jsblokland** [comment]
Here's one  example

aba8481e2d94d4ed4e0b7c3c11e51f0ac769d8ac30954bb48adc0128ee7a0eca

Verstuurd vanaf mijn iPhone

**Comment by jsblokland** [comment]
The transactions are still hanging there!
I recognize my wallet should have been updated but I think it's also fair to say that transactions getting stuck within the LoafWallet for more than a week also says something about the app as well. I tried sending litecoin to other apps and there were no problems. But the ones I send to LoafWallet cant be spend. 

So is there anyone who can help to get this solved? 


Verstuurd vanaf mijn iPhone

**Comment by losh11** [comment]
Is this for sending coins from Litecoin Core to LoafWallet. Transactions are likely not getting stuck within LoafWallet, if this is happening, please share transaction IDs and rescan the blockchain from within LoafWallet.

**Comment by jsblokland** [comment]
I updated to litecoin core. Than I used the 'rescan' prompt to rebuild the block chain. After that the balance showed a number that matched the number workout the transactions to LoafWallet. I then synchronized from within LoafWallet, but it still shows the transaction. They are now back to unverified after waiting for confirmation. Hence the synchronization does not had any effect on the balance in LoafWallet. I would expect them to disappear there.

aba8481e2d94d4ed4e0b7c3c11e51f0ac769d8ac30954bb48adc0128ee7a0eca

964e65b69e5d846d9e8fb8742e12282f3ed633538571355974ca606ab8c7df73



Verstuurd vanaf mijn iPhone
